core/src/test/java/com/mygdx/game/models/EliminationPhaseTest.java

```diff
@@ -0,0 +1,97 @@
+package com.mygdx.game.models;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+public class EliminationPhaseTest {
+    EliminationPhase ep1;
+    EliminationPhase ep2;
+    Brain b1;
+    Brain b2;
+    Brain b3;
+    int maxSelected;
+
+    @Before
+    public void setUp() throws Exception {
+        try {
+            b1 = new Brain();
+            b2 = new Brain();
+            b3 = new Brain();
+            maxSelected = 2;
+            ep1 = new EliminationPhase(new ArrayList<>(Arrays.asList(b1, b2, b3)), maxSelected);
+            ep2 = new EliminationPhase(new ArrayList<>(Arrays.asList(b1, b2, b3)));
+        }
+        catch (Exception e) {
+            throw new Exception("Failed to set up EliminationPhaseTest: ", e);
+        }
+    }
+
+    @Test
+    public void constructor() {
+        assertThrows("Should not be able to initialize with no brains",
+                IllegalArgumentException.class, () ->
+                        new EliminationPhase(new ArrayList<>(), maxSelected));
+        assertThrows("Should not allow null-brains",
+                IllegalArgumentException.class, () ->
+                        new EliminationPhase(new ArrayList<>(Arrays.asList(b1, null, b2)),
+                                maxSelected));
+    }
+
+    @Test
+    public void getBrains() {
+        assertEquals(Map.of(b1, false, b2, false, b3, false), ep1.getBrains());
```

Comment:
"of" does not work with my settings

Reply:
That's weird, what error message are you getting? And what's your system's java version?

Reply:
"Cannot resolve method 'of' in 'Map'" and no clue